### PR TITLE
PLANNER-828: Workbench: ScoreHolder support in GuidedRuleEditor shows error in server log

### PR DESCRIPTION
**Pull request by mcimbora**
All the accepted types are subclasses of ActionFieldList, refactor the code accordingly. Get rid of stinky ReflectiveVisitor.

@manstis Sounds good to you?

### CODE REVIEW

**Comment by baldimir** [comment]
+1 from me. Looks good.

File: drools-workbench-models/drools-workbench-models-commons/src/test/java/org/drools/workbench/models/commons/backend/rule/RHSClassDependencyVisitorTest.java [hunk 1]
```diff
@@ -0,0 +1,38 @@
+package org.drools.workbench.models.commons.backend.rule;
```
Comment (manstis) [inline drools-workbench-models/drools-workbench-models-commons/src/test/java/org/drools/workbench/models/commons/backend/rule/RHSClassDependencyVisitorTest.java:17]: License please.

**Comment by mcimbora** [comment]
@baldimir @manstis PR updated, thanks.

**Review by manstis — APPROVED** [review]

**Comment by manstis** [comment]
@mcimbora Please ping me for merge when green (again).

**Comment by mcimbora** [comment]
@manstis This looks good now. All green.

**Pull request merged**